js/webworkers/csv-export-worker.js

```diff
@@ -25,14 +25,29 @@ onmessage = function(event) {
             .join(opts.columnDelimiter);
     }
 
+    /**
+     * Converts `null` entries in columns and other empty non-numeric values to NaN value string.
+     * 
+     * @param {array} columns 
+     * @returns {string}
+     */
+    function joinColumnValues(columns) {
+        return _(columns)
+            .map(value => 
+                _.isNumber(value)
+                ? value
+                : "NaN")
```

Comment:
You probably want to emit NaN only for falsy values - if string is passed, it shall not be discarded 

```suggestion
                (_.isNumber(value) || _.value)
                ? value
                : "NaN")
```